Kitodo/src/test/java/de/sub/goobi/forms/ProjekteFormIT.java

```diff
@@ -74,4 +74,26 @@ public void shouldGetStatisticsManager() throws Exception {
         //assertEquals("Statistics Manager was not counted correctly!", Integer.valueOf(50), statisticsManager.getNumberOfPages());
         //assertEquals("Statistics Manager was not counted correctly!", Integer.valueOf(3), statisticsManager.getNumberOfVolumes());
     }
+
+    @Test
+    public void shouldDuplicateProject() throws Exception {
```

Comment:
This test belongs to `ProjectService` or `ProjectServiceIT` class as you test against the business logic and not the output of the form method. You can test both if you want as more tests are better then less tests.